backend/tests/prisonerQuickLook.test.ts

```diff
@@ -451,7 +458,7 @@ describe('prisoner profile quick look', () => {
               details: [
                 { label: 'Incentive level warnings', value: 0 },
                 { label: 'Incentive encouragements', value: 0 },
-                { label: 'Last incentive level review', value: '0 days ago' },
+                { label: 'Date of next review', html: 'Unable to show this detail' },
```

Comment:
I think _pretending_ that the last incentive review was "0 days ago" when data cannot be loaded was wrong. In my opinion we should state when we cannot show correct information.

Reply:
I think you're right about being more explicit (a user may see "0 days ago" and wrongly conclude the prisoner just had a reviewed).

This is when somehow the request to the API failed right?
I wonder if a better wording would be "Unable to retrieve this detail" - but maybe it's too low level?

Reply:
"Unable to show this detail" is the text already used all over this page and i think it's best to keep it consistent; i don't wanna make the call to rewrite this piece of content everywhere

Reply:
Oh there are other parts of DPS which use the exact same wording. Cool in that case consistency FTW :)